src/azul/indexer/aggregate.py

```diff
@@ -163,6 +167,30 @@ def get(self):
         return thaw(super().get())
 
 
+class DictAccumulator(Accumulator):
+    """
+    An accumulator of dictionaries. The key should return a unique value the
+    accumulation of dictionaries can be sorted by. (eg. `itemgetter('uuid')`)
+    """
+
+    def __init__(self, key: Callable):
+        self.key = key
+        self.value = {}
+
+    def accumulate(self, value):
+        require(isinstance(value, dict), value)
+        key = self.key(value)
```

Comment:
This use of self.key contradicts the documentation. Apparently the key is not just used for sorting.